UnityProjects/MRTKDevTemplate/ProjectSettings/ProjectSettings.asset

```diff
@@ -156,8 +156,8 @@ PlayerSettings:
   androidSupportedAspectRatio: 1
   androidMaxAspectRatio: 2.1
   applicationIdentifier:
-    Android: com.Microsoft.MRTK3Sample
-    Standalone: com.Microsoft.MRTK3Sample
+    Android: com.MixedRealityToolkitOrganization.MRTK3Sample
```

Comment:
should be org.MixedRealityToolkit  :)  For the registered MixedRealityToolkit.org domain.

Reply:
Overrode the auto-genned strings to match our domain